shop/controllers/feedback.py

```diff
@@ -11,7 +14,8 @@ def get_feedback_list(cls):
         return feedback
 
     @classmethod
-    def create_feedback(cls, author, product, title, content, images=None):
+    def create_feedback(cls, author, product, title, content, images=None, **_):  # images_to_delete arg is not
```

Comment:
what is it **_?

Reply:
images_to_delete arg comes here but it is not used in this method, so to avoid getting warnings I wrote this